src/test/java/core/basesyntax/shop/service/impl/ShopDataParserImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.shop.service.impl;
+
+import static core.basesyntax.shop.db.FruitShopStorage.getAll;
+
+import core.basesyntax.shop.dao.FruitShopDaoImpl;
+import core.basesyntax.shop.model.Fruit;
+import core.basesyntax.shop.service.ShopDataParser;
+import core.basesyntax.shop.strategy.FruitShopStrategyImpl;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ShopDataParserImplTest {
+    @AfterClass
+    public static void clearAfterTest() {
+        getAll().clear();
+    }
+
+    @Test
+    public void distribute_ParsingDataToStorage_ok() {
```

Comment:
```suggestion
    public void distribute_parsingDataToStorage_ok() {
```
please check all your test method names and make the naming consistent